src/networks.json

```diff
@@ -2222,5 +2222,45 @@
     },
     "start": 498623,
     "logo": "ipfs://QmSwCnEeLoyMApQd1YfhmGvLt97bH5mLPoabjKqUSuQHHF"
+  },
+  "9000": {
+    "key": "9000",
+    "name": "Evmos Network Testnet",
+    "shortName": "Evmos Testnet",
+    "chainId": 9000,
+    "network": "testnet",
+    "testnet": true,
+    "multicall": "0xbf84d77eb467c4aaa2988ac070cd119984c0aaa2",
+    "rpc": [
+      "https://eth.bd.evmos.dev:8545"
```

Comment:
These nodes don't look like working. ? not getting any response even after 5 mins. can take a look?

Reply:
Ah, our testnet was going through an upgrade. I can assure you they usually work (Blockdaemon hosts our public archive nodes)

![image](https://github.com/snapshot-labs/snapshot.js/assets/16395727/486c492f-6236-465e-80aa-9ea892977432)

Can you please check if it is still not working for you? 

One thing to note again, is that our chain does not start at block height 1 because of a hardfork, so the networks tool hosted on fleek will show an error.

Would love to bring Snapshot to the Cosmos ecosystem ASAP, thank you for the quick response!